src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/TaskTransformer.kt

```diff
@@ -20,13 +21,14 @@ class TaskTransformer(
   private val userTransformer: UserTransformer,
   private val risksTransformer: RisksTransformer,
   private val placementRequestTransformer: PlacementRequestTransformer,
+  private val workingDayCountService: WorkingDayCountService,
```

Comment:
@pezholio I think this is what is breaking the integration tests... I am assuming passing a service into a transformer is a no no or I'm not doing it right. Any tips? happy to do a call if easier

Reply:
I reckon you might have to add the workingDayCountService to the controller, and then pass the service to the transformer method itself. Something like:

```
@Service
class TasksController(
  private val userService: UserService,
  private val assessmentService: AssessmentService,
  private val placementRequestService: PlacementRequestService,
  private val taskTransformer: TaskTransformer,
  private val offenderService: OffenderService,
  private val placementApplicationService: PlacementApplicationService,
  private val enumConverterFactory: EnumConverterFactory,
  private val userTransformer: UserTransformer,
  private val taskService: TaskService,
  private val workingDayCountService: WorkingDayCountService,
) : TasksApiDelegate {

....

}
```

Then you can add the `workingDayCountService` to the transfomer method like so:

```
fun transformAssessmentToTask(assessment: AssessmentEntity, personName: String, workingDayCountService: WorkingDayCountService) 
```

And call it:

```
transformAssessmentToTask(assessment, personName, workingDayCountService)
```

in the controller.

Lemme know how you get on 👍

Reply:
thanks! I'll have a go

Reply:
yeah sure, I'm free from 1:30 tomorrow

Reply:
Cool - I've stuck an hour in the diary